tlslite/keyexchange.py

```diff
@@ -373,6 +373,9 @@ def calcVerifyBytes(version, handshakeHashes, signatureAlg,
                     SignatureScheme.ed448):
                 hashName = "intrinsic"
                 padding = None
+            elif signatureAlg[1] == SignatureAlgorithm.dsa:
+                hashName = HashAlgorithm.toRepr(signatureAlg[0])
+                padding = None
             elif signatureAlg[1] != SignatureAlgorithm.ecdsa:
```

Comment:
Nit: Wouldn't it be nicer by exchanging  the `!=` statement with `==` and swapping this statement's body with `else`'s ?

Reply:
no, there are four types here: eddsa, ecdsa, dsa, and rsa so a `!=` is shorter below